Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -419,6 +419,55 @@ public let CodeGenerators: [CodeGenerator] = [
         b.destruct(arr, selecting: indices, into: candidates, hasRestElement: probability(0.2))
     },
 
+    CodeGenerator("DestructObjectGenerator", input: .object()) { b, obj in
+        var properties = Set<String>()
+        for _ in 0...5 {
```

Comment:
Do we really always want to destruct 6 properties, not a random number between, say, 2 and 6?

Reply:
yeah could do, I'll update the range

Reply:
should we also do this for the op mutator as well?

Reply:
I think it's fine for the op mutator to just always change one property. Or do you mean something different?

Reply:
I meant the fixed loop that tries to find a new element:
```swift
for _ in 0...5 {
     let newElem = generator()
     ...
}
```
But now that I think about it, it shouldn't matter since we only need the selection to succeed once :)